runtime-modules/council/src/lib.rs

```diff
@@ -524,7 +532,17 @@ impl<T: Trait> Module<T> {
     }
 
     /// Checkout elected council members reward payments.
-    fn try_reward_payments(now: T::BlockNumber) {
+    fn try_process_budget(now: T::BlockNumber) {
+        // budget autorefill
+        let next_refill = NextBudgetRefill::<T>::get();
+        if next_refill == 0.into() {
```

Comment:
Am I correct to interpret this to say that the council module insists that the first refill happens at block heights `BudgetRefillPeriod*k` for integers `k>0`?. If so then I don't think we should enforce this, it should be up to the runtime developer to chose the initial value of `NextBudgetRefill` in the context of a deployment, and if this is occuring as part of a runtime upgrade, then it has to be done anyway, as block `0` based setup is never triggered.

Reply:
If you make `NextBudgetRefill` configurable + have a module setter function that runtime upgrade migration code can invoke, then I think that should be good enough.

Reply:
Yes, you are correct; this was an imperfect initialization mechanism. `NextBudgetRefill` and others made configurable + new extrinsic `plan_budget_refill` was created.